client-src/elements/chromedash-form-field.js

```diff
@@ -2,52 +2,66 @@ import {LitElement, html, nothing} from 'lit';
 import {ALL_FIELDS} from './form-field-specs';
 import {ref} from 'lit/directives/ref.js';
 import './chromedash-textarea';
-import {showToastMessage} from './utils.js';
+import {showToastMessage, getFieldValue} from './utils.js';
 
 export class ChromedashFormField extends LitElement {
   static get properties() {
     return {
       name: {type: String},
+      index: {type: Number}, // Represents which field this is on the form.
       stageId: {type: Number},
       value: {type: String},
-      index: {type: Number}, // Represents which field this is on the form.
+      fieldValues: {type: Array}, // All other field value objects
       disabled: {type: Boolean},
       checkboxLabel: {type: String}, // Optional override of default label.
       loading: {type: Boolean},
       fieldProps: {type: Object},
       forEnterprise: {type: Boolean},
       stageType: {type: Number},
       componentChoices: {type: Object}, // just for the blink component select field
+      checkMessage: {type: String},
     };
   }
 
   constructor() {
     super();
     this.name = '';
-    this.value = '';
     this.index = -1;
+    this.value = '';
+    this.fieldValues = [];
     this.checkboxLabel = '';
     this.disabled = false;
     this.loading = false;
     this.forEnterprise = false;
     this.stageType = undefined;
     this.componentChoices = {};
+    this.checkMessage = '';
+  }
+
+
+  getValue() {
+    return !this.value && this.fieldProps.initial ?
```

Comment:
Could this be `this.value || this.fieldProps.iniitial`?
Is there a logical difference?

Reply:
I copied that code that used to be in renderWidget, to be on the safe side.  I'm not sure what it should be, but what you suggest seems reasonable.  The logic might involve whether a value is 0 or null.  We should allow 0 values, so perhaps it should be:

  this.value != null ? this.value : this.fieldProps.initial

Reply:
OK, that looks like it would work.